packages/data-store/src/utils/contact/MappingUtils.ts

```diff
@@ -231,20 +264,74 @@ export const identityFrom = (identity: IdentityEntity): Identity => {
   }
 }
 
-export const metadataItemEntityFrom = (item: NonPersistedMetadataItem): IdentityMetadataItemEntity => {
-  const metadataItemEntity: IdentityMetadataItemEntity = new IdentityMetadataItemEntity()
-  metadataItemEntity.label = item.label
-  metadataItemEntity.value = item.value
+const metadataItemEntityFrom = <T extends MetadataTypes, U extends { new (): any }>(
```

Comment:
should we not export this function and make it available?

Reply:
It's not really meant to. There are the wrappers identityMetadataItemEntityFrom and contactMetadataItemEntityFrom wrappers to make the API more clear